### PR TITLE
pull a request to the origin fork ,master branch

### PR DESCRIPTION
**Pull request by zhouminghai**
pull a request to the origin fork ,master branch

### CODE REVIEW

File: branch1-file1 [hunk 1]
```diff
@@ -1,5 +1,7 @@
 branch1 file!
 
-haha
```
Comment (github-xiaogang) [inline branch1-file1:3]: why you delete my code ?

File: branch1-file1 [hunk 2]
```diff
@@ -1,5 +1,7 @@
...
+delete this line
+
```
Comment (github-xiaogang) [inline branch1-file1:5]: please delete this black line

File: zhoufile [hunk 3]
```diff
@@ -0,0 +1,7 @@
+create a file in a forked repo
+
+
```
Comment (github-xiaogang) [inline zhoufile:3]: this is good !!

File: branch1-file1 (hunk 1 above)
Comment (zhouminghai) [inline branch1-file1:3]: e.  I`m sorry , I`m not intended to do it

**Pull request merged**